### ISSUE TITLE
Allow for water input in ounces and mL

### ISSUE THREAD

**Issue by JI511** [JI511/Personal_Fitness#56]
This can be asked every time initially and then added into a saved preferences implementation.

**Comment by mdamonconnolly** [comment]
see issue #62 for handling of the saved preferences.
As for this issue as a whole, do you think `constants.py` should also contain conversation rates so that all values like water intake and other values (Here in the Uk we lift in kg, but I know in some places they use lbs) including features yet to be added, have a nice clean way to handle this, to (potentially) help keep any clutter out of the procedures.

**Comment by JI511** [comment]
Yes constants would be a great place to store these conversion formulas. Probably should make a function to do this and return the value.
